styleguide.config.js

```diff
@@ -109,6 +109,7 @@ module.exports = {
         {
           name: 'Forms',
           components: [
+            './packages/design-system/src/components/SearchBar/SearchBar.js',
```

Comment:
There is also a file named `setup.js` and `packages/design-system/src/components/index.js`. You need to add this new component there to make it work in external apps.